code/modules/language/vox.dm

```diff
@@ -1,7 +1,7 @@
 /datum/language/vox
 	name = "Vox-pidgin"
 	desc = "The common tongue of the various Vox ships making up the Shoal. It sounds like chaotic shrieking to everyone else."
-	key = "V"
+	key = "p"
```

Comment:
`p` is a key for Piratespeak, so that's not going to work either.

https://github.com/pariahstation/Pariah-Station/blob/88ab81e75b607cca8118d4b064b88039d6d994df/code/modules/language/piratespeak.dm#L1-L4

Reply:
Im going insane